Modules/_operator.c

```diff
@@ -1109,6 +1135,11 @@ static PyMethodDef itemgetter_methods[] = {
     {NULL}
 };
 
+static PyMemberDef itemgetter_members[] = {
+    {"__vectorcalloffset__", T_PYSSIZET, offsetof(itemgetterobject, vectorcall), READONLY},
```

Comment:
Why are you adding this attribute?

Reply:
Apparently this is the way to add vectorcall capability :disappointed: 
So, leave this and I'll open an issue to add the vectorcall capability a sensible way.